Lib/ssl.py

```diff
@@ -704,6 +704,15 @@ def create_default_context(purpose=Purpose.SERVER_AUTH, *, cafile=None,
     else:
         raise ValueError(purpose)
 
+    # `VERIFY_X509_PARTIAL_CHAIN` makes OpenSSL's chain building behave more
+    # like RFC 3280 and 5280, which specify that chain building stops with the
+    # first trust anchor, even if that anchor is not self-signed.
```

Comment:
I suggest adding an empty line for readability.

```suggestion
    # first trust anchor, even if that anchor is not self-signed.
    #
```